superset/config.py

```diff
@@ -984,7 +985,14 @@ def CSV_TO_HIVE_UPLOAD_DIRECTORY_FUNC(
 #    def SQL_QUERY_MUTATOR(sql, user_name, security_manager, database):
 #        dttm = datetime.now().isoformat()
 #        return f"-- [SQL LAB] {username} {dttm}\n{sql}"
-SQL_QUERY_MUTATOR = None
+def SQL_QUERY_MUTATOR(  # pylint: disable=invalid-name,unused-argument
```

Comment:
Which one is the `invalid-name`?

Reply:
@ktmud Pylint doesn’t like capitals in function names.